dnf/base.py

```diff
@@ -1282,7 +1282,7 @@ def _sig_check_pkg(self, po):
         if check:
             root = self.conf.installroot
             ts = dnf.rpm.transaction.initReadOnlyTransaction(root)
-            sigresult = dnf.rpm.miscutils.checkSig(ts, po.localPkg())
+            sigresult = dnf.rpm.miscutils.checkSig(ts, root, po.localPkg())
```

Comment:
Isn't this available as `ts.ts.rootDir` already?

Reply:
That's a good point, thank you.